packages/dapp-high-roller/src/components/app-game/app-game.scss

```diff
@@ -0,0 +1,10 @@
+.game {
+  padding: 1.5rem 1.5rem 0;
+}
+.actions {
+  display: flex;
+  margin-top: 1rem;
+}
+.actions > * + * {
```

Comment:
@Alonski Isn't this selector a bit too greedy?

Reply:
While I agree the CSS was taken from the HTML provided. I pretty much used it as is and broke it up into components. From what I understood from @ebryn I wasn't meant to spend too much time on styling and CSS.

Reply:
Nevermind this for now then :)